apps/web/src/config/experminetalFeatures.ts

```diff
@@ -0,0 +1,21 @@
+export enum EXPERIMENTAL_FEATURES {
+  WebNotifications = 'web-notifications',
+}
+
+export type FeatureRollOutConfig = {
+  feature: EXPERIMENTAL_FEATURES
+  percentage: number // For user, the roll out percentage is more intuitive and easy to understand. "coverage" may also be a good choice?
+  whitelist: string[] // Reason for this suggestion is, the user identifier is not necessarily to be the ip. We may switch to use things like device id or uuid to do the ab testing in the future
+}
+
+export type ExperimentalFeatureMpa = {
```

Comment:
naming suggestion. And also a typo
```suggestion
export type ExperimentalFeatureConfigs = {
```